js/game.js

```diff
@@ -7,36 +7,57 @@ import { Player } from './player.js';
 
 export class Game {
   constructor(inPlayers) {
```

Comment:
Not sure what this prefix `in` does for your parameter names? Are you using it as shorthand for `input`?